docs/source/include/examplenotes.rst

```diff
@@ -9,7 +9,7 @@ which should be installed via conda:
 
 ::
 
-   conda install -c astra-toolbox astra-toolbox
+   conda install astra-toolbox
```

Comment:
Why this change? ASTRA's docs say to use their channel (https://astra-toolbox.com/docs/install.html).